lib/news_scraper/configuration.rb

```diff
@@ -0,0 +1,16 @@
+module NewsScraper
+  class Configuration
+    DEFAULT_SCRAPE_PATTERNS_FILEPATH = File.expand_path('../../../config/article_scrape_patterns.yml', __FILE__)
+    attr_accessor :scrape_patterns, :scrape_patterns_filepath
+
+    def initialize(scrape_patterns_filepath: nil)
+      self.scrape_patterns_filepath = scrape_patterns_filepath if scrape_patterns_filepath
+    end
+
+    def scrape_patterns_filepath=(file_path)
```

Comment:
Stick all of this logic in the `initialize` method and remove the params from https://github.com/richardwu/news_scraper/pull/30/files#diff-739722dce01e765226d9e3872d8388fbR6 ?